Formula/erofs-utils.rb

```diff
@@ -0,0 +1,54 @@
+class ErofsUtils < Formula
+  desc "Utilities for Enhanced Read-Only File System"
+  homepage "https://git.kernel.org/pub/scm/linux/kernel/git/xiang/erofs-utils.git"
+  url "https://git.kernel.org/pub/scm/linux/kernel/git/xiang/erofs-utils.git/snapshot/erofs-utils-1.4.tar.gz"
+  sha256 "67702b1fc5da05719d95ddb7c107e334b04365f5161a9717479d2831fca85a98"
+  license "GPL-2.0-or-later"
+  head "git://git.kernel.org/pub/scm/linux/kernel/git/xiang/erofs-utils.git", branch: "master"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "lz4"
+  depends_on "util-linux" # for libuuid
+  on_linux do
+    depends_on "libfuse@2"
+  end
+
+  def install
+    system "./autogen.sh"
+    args = std_configure_args + %W[
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --prefix=#{prefix}
```

Comment:
```suggestion
    args = std_configure_args + %W[
      --disable-silent-rules
```
These are already in `std_configure_args`.

Reply:
Thanks, done.